### PR TITLE
Clear pause/away on controller button press

### PR DESCRIPTION
**Pull request by howard-stearns**

### CODE REVIEW

**Comment by howard-stearns** [comment]
https://app.asana.com/0/32622044445063/67945753172059

**Comment by howard-stearns** [comment]
Testing note: Don't get distracted by overlay placement (https://app.asana.com/0/search/89809419852408/89692255237085), which is a separate issue.

**Comment by ZappoMan** [comment]
:+1:

File: examples/away.js [hunk 1]
```diff
@@ -127,13 +148,28 @@ Controller.keyPressEvent.connect(function (event) {
...
         }
     }
+}
+
+Script.update.connect(maybeGoAway);
+// These two can be set now and left active as long as the script is running. They do not interfere with other scripts.
+Controller.mousePressEvent.connect(goActive);
+Controller.keyPressEvent.connect(maybeGoActive);
+// Set up the mapping, but don't enable it until we're actually away.
+eventMapping.from(Controller.Standard.LeftPrimaryThumb).to(goActive);
```
Comment (ZappoMan) [inline examples/away.js]: change these to add `.peek()` and you will find that you don't stop events for other mappings, in which case you can leave your mapping in...and not do the extra complexity of timers

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0d8404e7e8e1d0d51bc4661318fbd0e49f7a7820 available at: https://gist.github.com/f71aed14843322b86f0a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7086.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7086.dmg

**Comment by howard-stearns** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7c4e31687c79fe1fb5dc0bea9d1ead993e46085e available at: https://gist.github.com/f71aed14843322b86f0a<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7086.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7086.dmg

**Pull request merged**